system_tests/pubsub.py

```diff
@@ -185,12 +197,14 @@ def test_topic_iam_policy(self):
         topic_name = 'test-topic-iam-policy-topic' + unique_resource_id('-')
         topic = Config.CLIENT.topic(topic_name)
         topic.create()
-        count = 5
-        while count > 0 and not topic.exists():
-            time.sleep(1)
-            count -= 1
+
+        # Retry / backoff up to 7 seconds (1 + 2 + 4)
+        retry = RetryResult(lambda result: result, max_tries=4)
```

Comment:
This code wants 4 retries (to allow up to seven seconds) regardless of the default (which, after all, might change later and break this test sporadically).